README.md

```diff
@@ -540,11 +540,15 @@ See also [A comparison of operating systems written in Rust](https://github.com/
 
 [Rust Embedded](https://rust-embedded.org/)
 
+[awesome-embedded-rust](https://github.com/rust-embedded/awesome-embedded-rust)
```

Comment:
Can you please format this and the Rust Embedded link together? Not sure precisely how, but two random links doesn't feel like a good option (and the Rust Embedded one really shouldn't have just been dumped there like that to begin with). Maybe a sentence about the two, or another subheading list about general items or something?

Reply:
Addresses in #1288